test/hotspot/jtreg/serviceability/jvmti/vthread/MethodExitTest/libMethodExitTest.cpp

```diff
@@ -500,9 +500,6 @@ VirtualThreadUnmount(jvmtiEnv *jvmti, ...) {
   LOG("\nHit #%d: VirtualThreadUnmount #%d: enabling FramePop for method: %s::%s on virtual thread: %p\n",
          brkptBreakpointHit, ++vthread_unmounted_count, cname, mname, (void*)thread);
 
-  err = jvmti->NotifyFramePop(thread, 0);
-  check_jvmti_status(jni, err, "VirtualThreadUnmount: error in JVMTI NotifyFramePop");
-
```

Comment:
I assume this is needed as the result of your fix to return DUPLICATE for NotifyFramePop. Do we know that it always returns DUPLICATE for this particular call site?

Reply:
Yes, it is related to the DUPLICATE change.
The test calls JVMTI `NotifyFramePop` in the `VirtualThreadMount`/`VirtualThreadUnmount` event handlers.
One of them is a dup for sure and not needed.